docs/config-ha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 20
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which can be enabled after Knative Serving control plane is installed.
```

Comment:
I was surprised these docs were pretty serving-centric given their location and the title.

e.g. How do I do the same for eventing?

Reply:
There is another page that is under serving but applies to eventing as well: https://knative.dev/docs/serving/accessing-metrics/

Should we move this under "Installing Knative" (which I think should be renamed to "Getting Started with Knative")?

Reply:
I'd rather not move that more prominently as it builds on something we're turning down :)

Reply:
Fair, I wasn't sure where to put it. I'll move it under Serving until we have something related to HA for Eventing.